README.md

```diff
@@ -6,46 +6,23 @@
 
 As of August 14, 2020, the [EPUB 3 Working Group](https://www.w3.org/publishing/groups/epub-wg) is now the maintainer of the EPUB 3 specifications.
 
-## EPUB 3.3
+## Current version of EPUB: EPUB 3.3
 
-Editor's Drafts of documents under active development:
+On the 25th of May, 2023, the [EPUB 3 Working Group](https://www.w3.org/publishing/groups/epub-wg) has published three 
+EPUB 3.3 Recommendations ([EPUB 3.3](https://www.w3.org/TR/epub-33/), [EPUB Reading Systems 3.3](https://www.w3.org/TR/epub-rs-33/), and [EPUB Accessibility 1.1](https://www.w3.org/TR/epub-a11y-11/)), as well as a number of Working Group Notes. For a complete list of the publications by the Working Group, see the
+[separate index page](https://www.w3.org/publishing/epub33/).
 
-- [EPUB 3 Overview](https://w3c.github.io/epub-specs/epub33/overview/)
-- [EPUB 3.3](https://w3c.github.io/epub-specs/epub33/core/) (On Recommendation track)
-- [EPUB Reading Systems 3.3](https://w3c.github.io/epub-specs/epub33/rs/) (On Recommendation track)
-- [EPUB Structural Semantics Vocabulary 1.1](https://w3c.github.io/epub-specs/epub33/ssv/)
-- [EPUB 3 Text-to-Speech Enhancements 1.0](https://w3c.github.io/epub-specs/epub33/tts/)
-- [EPUB Accessibility 1.1](https://w3c.github.io/epub-specs/epub33/a11y/) (On Recommendation track)
-- [EPUB Accessibility Techniques 1.1](https://w3c.github.io/epub-specs/epub33/a11y-tech/)
-- [W3C EPUB Accessibility - EU Accessibility Act Mapping](https://w3c.github.io/epub-specs/epub33/epub-a11y-eaa-mapping/)
+The Working Group maintains an [errata](https://w3c.github.io/epub-specs/epub33/errata.html) file for all the specifications and notes. If you want to report a new
+erratum, please follow the guidelines in that file.
 
-
-The EPUB 3.3 revision started on September 11, 2020. It is [chartered](https://www.w3.org/2020/08/epub-wg-charter.html) to run until August 2023.
-
-For more information about the meeting schedule, how to join or participate, etc., please consult the [group's home page](https://www.w3.org/publishing/groups/epub-wg), or contribute to this repository via issues and comments.
-
-## EPUB 3 Current Version
-
-The current version of EPUB 3 is [EPUB 3.2](https://www.w3.org/publishing/epub32/).
-
-This version was published on May 8, 2019, by the former EPUB 3 Community Group.  
-
-## Open Issues
-
-The following links provide filtered views of the currently open issues:
-
-- [EPUB 3 core specifications](https://github.com/w3c/epub-specs/issues?q=is%3Aissue+is%3Aopen+-label%3ACat-NonCore+-label%3ACat-Accessibility+)
-- [EPUB Accessibility specification](https://github.com/w3c/epub-specs/issues?q=is%3Aissue+is%3Aopen+label%3ACat-Accessibility+)
-
-Note: Issues for specifications not actively maintained have now been [marked as abadoned and closed](https://github.com/w3c/epub-specs/issues?q=is%3Aissue+label%3AStatus-Abandoned+is%3Aclosed). These issues may be resurrected in the future if work is resumed on their respective specifications.
```

Comment:
I only wonder if we should keep this note around so it's not forgotten that we did this. Otherwise, I'm fine with the changes. (I assume we'll update it again whenever the maintenance group is approved.)

Reply:
You are right. And yes, we will have to update this when the PM WG is approved and started.